packages/ra-auth-cognito/src/ErrorMFARequired.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Error thrown when a user is required to enter a MFA code from SMS.
+ * Unsupported.
+ */
+export class ErrorMFARequired extends Error {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** IMO we should treat this error like the others which show a readable message to the end user. Even if the SMS functionality is not set. Otherwise how can user and developers know that this error is raised, other than by reading the console? It's a bit hazy I think.

<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏At the same time, since this error concerns SMS and to be consistent with the other errors, we could rename it like `ErrorMfaSmsRequired`‏. Note the letter case of `Mfa`.

```suggestion
export class ErrorMfaSmsRequired extends Error {
```